src/quo2/screens/main.cljs

```diff
@@ -90,7 +91,10 @@
                  :component channel/preview-channel}
                 {:name      :preview-lists
                  :insets    {:top false}
-                 :component preview-lists/preview-preview-lists}]
+                 :component preview-lists/preview-preview-lists}
+                {:name      :page-nav
```

Comment:
this is the wrong section also. it should be in navigation, right?

Reply:
Done